tests/php/Forms/SearchableDropdownTraitTest.php

```diff
@@ -168,6 +168,18 @@ public function provideGetValueArray(): array
                 'value' => '-1',
                 'expected' => [],
             ],
+            'array - empty' => [
+                'value' => [],
+                'expected' => [],
+            ],
+            'array - empty string' => [
+                'value' => [''],
+                'expected' => [],
+            ],
+            'array - zero string' => [
+                'value' => ['0'],
+                'expected' => [],
+            ],
         ];
```

Comment:
What happens if the value is `[1]` or `['1']`?

Reply:
I’ve pushed a couple of new values to the data provider to find out 😉